package.json

```diff
@@ -41,14 +43,21 @@
     "preact": "^10.0.4",
     "purify-css": "^1.2.5",
     "purifycss-webpack": "^0.7.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "sass": "^1.23.7",
     "sass-loader": "^8.0.0",
     "style-loader": "^1.0.0",
     "url-loader": "^2.2.0",
+    "utils": "^0.3.1",
     "webpack": "^4.41.2",
     "webpack-bundle-size-analyzer": "^3.1.0",
     "webpack-cli": "^3.3.10",
     "webpack-dev-server": "^3.9.0",
     "webpack-hot-middleware": "^2.25.0",
     "yaml-loader": "^0.5.0"
+  },
+  "dependencies": {
+    "currentscript": "^1.1.1"
```

Comment:
Was this missing? I don't see `currentscript` being used in the new code.

Reply:
Don't really know, but I think it is required `current-executing-script`

Reply:
fixed with: https://github.com/mkkeck/coi/commit/f834bb0f0e6b18f91552cd99063337d56f1b97ec